MyCryptoHQ/MyCrypto
# MyEtherWallet -> MyCrypto

Closes #967. The king is dead, long live the king. Changes everything MEW to MyCrypto. Everything in translations was done via find -> replace, but all of the code changes were done by hand to assure that everything's kosh 👌  Here's a full list of the changes:

### Changes

* All text appearances of MyEtherWallet / MEW have been changed to MyCrypto
    * I by and large did not use "MyC" anywhere since it's a little more confusing, and MyCrypto is quite short anyway.
* Updated all visual assets to use new MyCrypto logo / name
* Removed a ton of unused visual assets (R.I.P. mew and mewtwo)
* Removed all favicons, in favor of generating everything from a single `favicon.png` in common/assets/images via the webpack favicons plugin.
    * We were already using this, not the other favicons in the project, so this isn't much of a change.
* All help links point at support.mycrypto.com, and have been converted from NewTabLink to a HelpLink component to make future refactoring easier, and help enforce correct URLs.

### Things not changed / Unknowns

* ~Will we point people to a public slack anymore? If so, or if it's a different chat, what's the info?~ Removed, since it's not on the support footer
* ~Subreddit / Facebook / LinkedIn / Mailchimp URLs unchanged~ - Updated everything except Mailchimp
* ~Are the Trezor, Ledger, and DigitalBitbox referral URLs getting updated?~ Updated from support
* ~Should I still point people to the MEW chrome extension? Are we renaming / posting a new one / deprecating?~ Removed, since it's not in the support footer
* ~Do we have new donation addresses?~ Updated from support site
* Bity has a `mew@bity.com`, are we going to change that?

^ I've messaged @tayvano about all of the above, just wanted to mark them on the PR and keep it in WIP until changed.

### Known Issues

* You may not see the favicon update in dev, because favicon generation was removed from dev to speed it up. However, building the HTML downloadable version will show it.
* The icons use the primary brand color from before, #0e97c0, despite the logo assets using #007896. @tayvano confirmed this was some kind of CMY -> RGB export bug, and will be updating assets accordingly.

### Steps to Test

* Browse around the site and ensure you don't see "MyEtherWallet" or "MEW" in any text
* Confirm that all emails links are sent to @mycrypto.com
* Confirm that all logos seen use the MyCrypto icon and say "MyCrypto"
* Confirm that all help links go to support.mycrypto.com
* Go through Onboarding and ensure all of the above


### Screenshot

<img width="876" alt="screen shot 2018-02-01 at 3 58 37 pm" src="https://user-images.githubusercontent.com/649992/35703064-dbc89dd2-0768-11e8-9dc1-202e3d85488f.png">
